### DISCUSSION TITLE
Gigabyte X399 DESIGNARE EX (rev. 1.0)

### DISCUSSION THREAD

**Discussion by greyltc** [xCuri0/ReBarUEFI#40] [General]
Should I expect any of this to work on my motherboard?

https://www.gigabyte.com/uk/Motherboard/X399-DESIGNARE-EX-rev-10/support#support-cpu

**Comment by xCuri0** [comment]
Why not try and see ?  There is no reason it shouldn't

**Reply by greyltc** [reply]
Yeah, I probably will try it if I can't convince gigabyte to generate an official beta bios that does it. Still though, flashing a manually modified BIOS doesn't feel like a risk-free procedure to me. It looks like most of the success here has been with Intel chipsets. Are you aware of any success stories with an X399-based motherboard?

**Comment by xCuri0** [comment]
No but people have had it work on all kinds of hardware even server boards
